runtime/libc/wasi/wasi_impl_uvwasi.c

```diff
@@ -0,0 +1,315 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+#include <uvwasi.h>
```

Comment:
I'm just going to assume this is a straightforward port of uvwasi and do a very light scan.